libfreerdp/codec/bitmap.c

```diff
@@ -435,8 +435,8 @@
 #define TEST_MIX \
 		((last_line == 0 && pixel == mix) || \
 				(last_line != 0 && pixel == (ypixel ^ mix)))
-#define TEST_FOM (TEST_FILL || TEST_MIX)
-#define TEST_COLOR (pixel == last_pixel)
+#define TEST_FOM TEST_FILL || TEST_MIX
```

Comment:
In fact it's not really a gain but with -Wparentheses-equality there are some warnings but I think it's worth to have the warnings enabled in general.